### ISSUE TITLE
Audio burrito flavor types seem to give too much information

### ISSUE THREAD

**Issue by gtryus** [bible-technology/scripture-burrito#317]
[Audio Burrito flavor details](https://docs.burrito.bible/en/latest/flavors/scripture_audio_flavor.html#metadata-flavor-details) lists the bitRate, bitDepth, and samplingRate as optional parameters. The bitRate, for example doesn't really work for compressed audio such as mp3. Is it better to list the compression type as "mp3" or to list it using the mimeType like the ingredients does: "audio/mpeg". There doesn't seem to be any way to link ingredients to a particular format ... which seems like it would be necessary if there were multiple formats. Would we have multiple formats of wav or mp3 files in one burrito or should we just use the mimeType in the ingredient to reference the formats in the flavor type?

I think the committee also decided we would like to add "ogg" or the mimeType: "audio/ogg;codecs=opus" to the list of supported types. Should we add "m4a" which is what many cell phones create when they record? Do we need the limit or should we just support a larger list?

**Comment by jsarabia** [comment]
I agree that mimetype would make more sense than compression (especially with the compression field being container types). Is it best to have the codec be in the mimetype, or should that be a separate optional field (and just leave it "audio/ogg" in the example)?

I would like the simplicity of the formats matching on mimetype. It might be valuable to have a variety of formats within a burrito such as wav, mp3. I suppose if you wanted to offer various qualities of compressed formats like mp3 though, you would need to have multiple formats for a mimetype (and then a way for an ingredient to link to it).

**Comment by jtauber** [comment]
The Audio Burrito flavor details were inherited and haven't been discussed since I joined. Happy to rethink them all.

**Comment by jtauber** [comment]
App developers to weigh in on what properties they actually need for each file.

What properties can we drop from the audio flavor?

**Comment by jtauber** [comment]
- move properties to individual ingredients
- consider: tag array for each ingredient or arbitrary JSON object or string key to string value mapping
- at overall SB spec level, ingredients can have a JSON object "properties"
- controlled vocabulary keys ("x-" for extension)
- "properties" are optional for audio files

**Comment by jtauber** [comment]
change "reading" to "nonDrama"

**Comment by jtauber** [comment]
To summarize / clarify the changes I'm making to the spec:

- ingredients to have an optional JSON object "properties"
- keys are either in the spec or start with "x-"
- remove "formats" and "conventions" (but then is anything left in the flavour?)
- only property retained is "performance" and that's moved to ingredient properties
- but change "reading" to "nonDrama"

**Comment by gtryus** [comment]
@jonathanrobie This issue should be labeled as Documentation or perhaps Implementation (or both).
